lib/gemfury/command/app.rb

```diff
@@ -198,23 +198,23 @@ def git_list
     end
   end
 
-  desc "git:rename", "Rename a Git repository"
+  desc "git:rename NAME", "Rename a Git repository"
```

Comment:
Should be `git:rename OLD_NAME NEW_NAME`

```suggestion
  desc "git:rename OLD_NAME NEW_NAME", "Rename a Git repository"
```

or maybe the following, for consistency with others:

```suggestion
  desc "git:rename REPO_NAME NEW_NAME", "Rename a Git repository"
```